apps/docs/src/app/examples/switches/SwitchWithIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import {
+  HvBaseSwitch,
+  HvButtonBase,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <HvButtonBase
+      className="relative items-center rounded-full [&_.HvBaseSwitch-root]:bg-transparent"
```

Comment:
this can't be a button. also, what are the `.HvBaseSwitch-root` styles for? and why can't they be on the BaseSwitch?

Reply:
The `BaseSwitch` has it's own hover styles which are being compounded with the `ButtonBase` hover styles (as they use alpha).